core/src/apps/management/reset_device/layout.py

```diff
@@ -184,11 +208,8 @@ async def slip39_basic_show_and_confirm_shares(
             await show_share_words(ctx, share_words, index)
 
             # make the user confirm words from the share
-            if await _confirm_share_words(ctx, index, share_words):
-                await _show_confirmation_success(ctx, index, len(shares))
+            if await _share_words_confirmed(ctx, index, share_words):
```

Comment:
looks like this should be `_share_words_confirmed(ctx, index, share_words, len(shares))` ?

Reply:
Correct, changed that in [2775364](https://github.com/trezor/trezor-firmware/pull/2741/commits/27753645e6ba1e4f925f28baeb25e78717936ffd) - together with recording the fixtures